test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/BindingSourceModelBinderTest.cs

```diff
@@ -84,12 +84,12 @@ public void BindingSourceModelBinder_ThrowsOnNonGreedySource()
 
             var binder = new TestableBindingSourceModelBinder(BindingSource.Body);
 
-            // Act 
+            // Act
             var result = await binder.BindModelAsync(context);
 
             // Assert
             Assert.NotNull(result);
-            Assert.True(result.IsModelSet);
+            Assert.False(result.IsModelSet);
             Assert.True(binder.WasBindModelCoreCalled);
```

Comment:
While you're doing cleanup here can you rename these tests to be more accurate? `BindingSourceModelBinder_ReturnsTrue_MatchingSource` `BindingSourceModelBinder_ReturnsFalse_NonMatchingSource`

They don't really return true or false